Makefile

```diff
@@ -46,14 +46,17 @@ endif
 #----------------------------------------------------------------------
 
 # Explicitly set the default target to do everything that isn’t already done
-default: | init assets all public ;
+default: | init lint assets all public ;
 
 # Run anything that needs doing post-checkout to make this buildable
 init: node_modules ;
 
 # Use building English language as a check to see if everything works
 test: en ;
 
+lint:
+	find source -type f -name '*.json' -print -exec jsonlint -q '{}' \;
```

Comment:
This probably doesn't need `-print` (does `jsonlint` not say what it's working on anyway?), and I'd always reach for `-execdir` unless the linter specifically can't handle that. Probably not a big deal in this case but it should be the default unless there is a reason it needs to run from the project root.